app/content/views/leaderboard.py

```diff
@@ -0,0 +1,48 @@
+from django.db.models import Count
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import filters, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import BasePermission
+
+from app.common.mixins import ActionMixin
+from app.common.pagination import BasePagination
+from app.common.permissions import IsMember
+from app.content.filters.leaderboard import LeaderBoardFilter
+from app.content.models.user import User
+from app.content.models.user_badge import UserBadge
+from app.content.serializers.badge import BadgeSerializer
+from app.content.serializers.user_badge import LeaderboardSerializer
+
+
+class LeaderboardViewSet(viewsets.ModelViewSet):
+    queryset = (
+        User.objects.all()
+        .annotate(number_of_badges=Count("user_badges"))
+        .order_by("-number_of_badges")
```

Comment:
Du trenger ikke annontere her i tillegg til å kjøre en egen spørring for det samme i serializeren

Reply:
Tanken med annotate er at man kan sortere basert på et felt som ikke kommer før i serializeren. Er det en annen bedre måte å sortere?

Reply:
Ahh, skjønner. Har ikke sånn veldig mye erfaring med slik sortering men kanskje @Eirsteir , @norgor , @Her0elt eller @trthingnes har noen tanker om hva som er lurt her.